scripts/minikube_setup.sh

```diff
@@ -14,14 +14,26 @@ set -e
 : ${KUBERNETES_VERSION:=v1.21.2}
 
 OS=$( uname -s | tr '[:upper:]' '[:lower:]' )
+ARCH=$( uname -m | sed 's/x86_64/amd64/')
```

Comment:
```
UNAME=$( uname -m)
case $ARCH in
   x86_64)
     ARCH="amd64"
     ;;
   aarch64)
     ARCH="arm64"
     ;;
   *)
     ARCH="amd64"
     ;;    
esac
```

Reply:
> ```
> UNAME=$( uname -m)
> case $ARCH in
>    x86_64)
>      ARCH="amd64"
>      ;;
>    aarch64)
>      ARCH="arm64"
>      ;;
>    *)
>      ARCH="amd64"
>      ;;    
> esac
> ```

Thanks, I will change it to this. One small issue is that code will convert `arm64` into `amd64` due to that final case, so if the ARCH gets reported as arm64 then it'll fail. (I'll fix that when I update my PR)

Reply:
Updated the scripts. Note that I removed the `*)` case as that causes `arm64` to get incorrectly converted into `amd64`.